docs/guide/ecosystem/server-side-rendering.md

```diff
@@ -119,6 +119,9 @@ If you are searching for a well tested solution, with a large community and nice
 
 Learn more about it on the dedicated [Nuxt](./nuxt.md) page.
 
+## Servue
+[Servue](https://futureaus.github.io/servue/) shares a similar scope to Nuxtjs, It allows you to render `.vue` files into rendered html. It includes features for head management, layouts and more. Compared to nuxt, it is more suitable for Multi-page-applications using existing server/routing systems such as koa/koa-router and express.js
```

Comment:
```suggestion
[Servue](https://futureaus.github.io/servue/) shares a similar scope to Nuxtjs, it allows you to render `.vue` files into rendered html. It includes features for head meta tag management, layouts and more. Servue is more suitable for Multi-page-applications using existing server-side routing systems such as koa/koa-router or express.js
```

Reply:
I would also advice you to write a tutorial in the future on how to use it, then you can link it here. People like to see it in action.

Reply:
All the documentation is found on that page, I don't really have any tutorials or videos about it, and haven't had any people having issues with the package since it's pretty straight forward imo

Reply:
Alright. If you are OK with the proposed changes apply them and I will merge it :)

Reply:
Great.

Reply:
The changes have not been applied, just resolved. There are grammatical errors to be fixed before merging. Please either use what I wrote or fix them yourself so we can merge this :)

Reply:
Oh, should be done now.